src/components/ExpandCollapse/ExpandCollapse.jsx

```diff
@@ -43,6 +43,7 @@ const ExpandCollapse = forwardRef(
               iconSize={iconSize}
               tabindex="-1"
               clickable={false}
+              ignoreFocusStyle
```

Comment:
this is the wrong fix. I saw you need to pass it even clickable false so I understand the issue is that the accordion icon have focus even it is not a button (clickable=false).
the reason for the focus beahvior which we don't want to allow is tabindex="-1".
we must remove it because we do not want to allow focus on only decorative element (if you don't sure why this is the case let's talk)

Reply:
Let's talk about this one to clarify a bit

Reply:
Removed both `tabIndex` and `ignoreFocusStyle`